src/runtime/particle-execution-host.ts

```diff
@@ -111,6 +111,16 @@ export class ParticleExecutionHost {
     apiPort.InstantiateParticle(particle, particle.id.toString(), particle.spec, stores);
   }
 
+  reinstantiate(particle: Particle, /*pecId: String,*/ stores: Map<string, StorageProviderBase>): void {
```

Comment:
Delete inline comment? Or otherwise add a TODO

Reply:
this is a leftover, sorry, deleted.